ihmc-perception/src/main/java/us/ihmc/perception/gpuHeightMap/RapidHeightMapExtractorCUDA.java

```diff
@@ -0,0 +1,529 @@
+      sensorCroppedTerrainCostImage.close();
+      sensorCroppedContactMapImage.close();
+   }
+
+   public int getCenterIndex()
+   {
+      return centerIndex;
+   }
+
+   public void populateParameterBuffers(HeightMapParameters parameters, CameraIntrinsics cameraIntrinsics, Tuple3DReadOnly gridCenter)
+   {
+      paramsArray = new float[] {(float) parameters.getLocalCellSizeInMeters(),
+                                 (float) centerIndex,
+                                 (float) cameraIntrinsics.getHeight(),
+                                 (float) cameraIntrinsics.getWidth(),
+                                 (float) gridCenter.getX(),
+                                 (float) gridCenter.getY(),
+                                 (float) mode,
+                                 (float) cameraIntrinsics.getCx(),
+                                 (float) cameraIntrinsics.getCy(),
+                                 (float) cameraIntrinsics.getFx(),
+                                 (float) cameraIntrinsics.getFy(),
+                                 (float) parameters.getGlobalCellSizeInMeters(),
+                                 (float) globalCenterIndex,
+                                 (float) parameters.getRobotCollisionCylinderRadius(),
+                                 gridOffsetX,
+                                 (float) parameters.getHeightFilterAlpha(),
+                                 (float) localCellsPerAxis,
+                                 (float) globalCellsPerAxis,
+                                 (float) parameters.getHeightScaleFactor(),
+                                 (float) parameters.getMinHeightRegistration(),
+                                 (float) parameters.getMaxHeightRegistration(),
+                                 (float) parameters.getMinHeightDifference(),
+                                 (float) parameters.getMaxHeightDifference(),
+                                 (float) parameters.getSearchWindowHeight(),
+                                 (float) parameters.getSearchWindowWidth(),
+                                 (float) cropCenterIndex,
+                                 (float) parameters.getMinClampHeight(),
+                                 (float) parameters.getMaxClampHeight(),
+                                 (float) parameters.getHeightOffset(),
+                                 (float) parameters.getSteppingCosineThreshold(),
+                                 (float) parameters.getSteppingContactThreshold(),
+                                 (float) parameters.getContactWindowSize(),
+                                 (float) parameters.getSpatialAlpha(),
+                                 (float) parameters.getSearchSkipSize(),
+                                 (float) parameters.getVerticalSearchSize(),
+                                 (float) parameters.getVerticalSearchResolution(),
+                                 (float) parameters.getFastSearchSize()};
+
+      initialized = true;
```

Comment:
Is this actually used? I couldn't find any usages of initialized or isInitialized()